src/widget/weffect.cpp

```diff
@@ -44,12 +44,13 @@ void WEffect::effectUpdated() {
         EffectPointer pEffect = m_pEffectSlot->getEffect();
         if (pEffect) {
             const EffectManifest& manifest = pEffect->getManifest();
-            if (! manifest.shortName().isEmpty()) {
+            if (!manifest.shortName().isEmpty()) {
                 name = manifest.shortName();
+                description = manifest.name() + ": " + manifest.description();
```

Comment:
French typography rule is simple (this is the only simple thing in french): if the punctuation mark is made of a single part (like dot, comma, ...), there is no space before and one space after. If it is made of two parts (like colon, semicolon, ...), there is a non-breaking space before and a space after.
with the exception of parenthesis, brackets, ... which have a space outside and no space inside.